### PR TITLE
Abstract raw files read as a util function

### PR DESCRIPTION
**Pull request by shifucun**
This is going to be used in server module as well

### CODE REVIEW

File: tool/src/main/java/org/datacommons/tool/Lint.java [hunk 1]
```diff
@@ -54,56 +54,23 @@
...
-        mcfFiles.size(),
-        tmcfFiles.size(),
-        csvFiles.size());
-    // Various checks
-    if (nTsv > 0 && nTsv != csvFiles.size()) {
-      throw new CommandLine.ParameterException(
-          spec.commandLine(), "Please do not mix .tsv and .csv files");
-    }
+
+    FileGroup fg = TmcfCsvReader.read(files, spec, logger);
```
Comment (pradh) [inline tool/src/main/java/org/datacommons/tool/Lint.java:58]: The class TmcfCsvReader gives the impression that it is actually reading the files, but it is just enumerating them. Moreover, this exists to just build a FileGroup, so how about making it a static function inside FileGroup?  

FileGroup.Build(...)

**Review by pradh — COMMENTED** [review]
Nice refactor.

File: tool/src/main/java/org/datacommons/tool/Lint.java (hunk 1 above)
Comment (shifucun) [reply tool/src/main/java/org/datacommons/tool/Lint.java]: Yea, that's better

**Review by pradh — APPROVED** [review]
Thanks.

**Pull request merged**